tests/kuttl/test-sidecars/01-assert.yaml

```diff
@@ -37,4 +38,5 @@ spec:
         spec:
           containers:
           - name: puptoo-cron
-          - name: token-refresher
\ No newline at end of file
+          - name: token-refresher
```

Comment:
I know it's an extra ask - but could we also have an assertion of the image here from the config? I'm not asking to test all versions, but just the default case.